jekyll/_cci2/container-runner.adoc

```diff
@@ -475,96 +475,38 @@ While jobs that run with container-agent cannot use CircleCI's <<building-docker
+  spec:
+    runtimeClassName: sysbox-runc
+    ...
 ```
 
-This will now let you run Buildah commands with container agent jobs and build containers:
+With sysbox installed and the runtime class updated in your container-agent resource class configuration, you may now use Sysbox in your `.circleci/config.yml` syntax. example:
 
 ```yaml
-  docker-image:
-    docker:
-      - image: quay.io/buildah/stable
-    resource_class: <namespace>/<resourceClass>
-    steps:
-      - checkout
-      - run:
-          name: sanity-test
-          command: |
-            buildah version
-      - run:
-          name: Building-a-container
-          command: |
-            buildah bud -f ./Dockerfile -t myimage:0.1
-            buildah push myimage:tag
-```
-
-[#using-buildah-with-custom-images]
-=== Using Buildah with custom images
-
-You can also build your own custom image and include the installation of Buildah in your Dockerfile:
-
-```
-sudo yum install buildah
-```
-
-If you plan to use a CircleCI link:https://circleci.com/developer/images[convenience image], ensure you add the repository for installation to your job's `steps`:
-
-```
-sudo apt-get update
-sudo apt-get install -y wget ca-certificates gnupg2
-VERSION_ID=$(lsb_release -r | cut -f2)
-echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel-kubic-libcontainers-stable.list
-curl -Ls https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_$VERSION_ID/Release.key | sudo apt-key add -
-sudo apt-get update
-sudo apt install buildah -y
-```
-
-Additionally, set the isolation variable to default to `chroot`:
-
-```
-# Default to isolate the filesystem with chroot.
-ENV BUILDAH_ISOLATION=chroot
+docker:
+  - image: registry.nestybox.com/nestybox/ubuntu-bionic-systemd-docker
+    entrypoint: ["dockerd"]
+    command: ["--host=unix:///var/run/docker.sock"]
 ```
 
-You can then follow the same instructions as <<#using-the-buildah-image,Using the Buildah image>> above to add the fuse device plugin to the container-agent deployment and update your `.circleci/config.yml` file to use your custom images and build container images in those jobs.
-
 [#limitations]
 == Limitations
 
 * Any known <<runner-overview#limitations,limitation>> for the existing self-hosted runner will continue to be a limitation of container agent.
 * Only Kubernetes container environments are supported at this time.
 * <<building-docker-images#,`setup_remote_docker`>> as a command is not supported with container runner.  See <<#building-container-images,Building Container Images>>.
 * `aws_auth.oidc_role_arn` is not supported on the container runner. You can set up AWS authentication using the `aws_auth` field. More information can be found in the xref:configuration-reference#oidc[Configuration Reference].
+* Sysbox requires an Ubuntu based image with Containerd.
```

Comment:
Is there documentation on this? I wasn't aware of Sysbox _requiring_ an Ubuntu-based image.

Reply:
https://github.com/nestybox/sysbox?tab=readme-ov-file#using-sysbox

> You can choose whatever container image you want, Sysbox places no requirements on the image.

Reply:
:wisdomdog:

Reply:
That said, for the purposes of DinD and other system-level stuff, we should document that.

Reply:
@christian-stephen here: https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-k8s.md#kubernetes-worker-node-requirements
it lists ubuntu as a requirement for k8s worker nodes, but I do see in other sections where it seems to imply that it should work everywhere. still above this section in the same doc they basically only tested on ubuntu to.

Reply:
done!

Reply:
Thanks!